server/index.js

```diff
@@ -141,36 +157,54 @@ function logRooms() {
 
 function showVotes(roomId) {
     const roomTickets = tickets.filter(p => p.roomId == roomId);
-
+    // find the text in the gametype where the index is the closest
+    let closest = 0;
+    const average = getAverage(roomId);
     if (roomTickets) {
-        const average = getAverage(roomId);
-        const fib = [0, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
-        let closest = 0;
+        const fib = gameType.find(p => p.roomId == roomId).gameType.values
+        closest = 0;
         let smallestDiff = Number.MAX_VALUE;
-        for (const number of fib) {
+        for (const item of fib) {
+            // get the index of the item in the fib array
+            const number = fib.indexOf(item);
             const difference = Math.abs(number - average);
-            if (difference < smallestDiff) {
+            if (difference <= smallestDiff) {
                 smallestDiff = difference;
                 closest = number;
             }
         }
-
         const ticket = roomTickets.find(f => f.votingOn);
         if (ticket) {
-            ticket.score = closest;
+            ticket.score =  gameType.find(p => p.roomId == roomId).gameType.values[closest];
         }
     }
 
-    io.to(roomId).emit('show');
+    const nearest = gameType.find(p => p.roomId == roomId).gameType.values[closest];
```

Comment:
Updated the code so that we calculate the values in the back end and then pass it to the UI instead of letting the UI do the calculations.